packages/foam-vscode/static/dataviz/index.html

```diff
@@ -23,6 +19,6 @@
       4. open this file in a browser
     -->
     <!-- <script src="./test-data.js"></script> -->
-    <script data-replace src="./graph.js"></script>
+    <script replace-data src="./graph.js"></script>
```

Comment:
to be valid html data attributes need to start with `data-` https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
Oops didn't realize.